### ISSUE TITLE
podcast compact view

### ISSUE THREAD

**Issue by kylepolich** [data-skeptic/dataskeptic.com#683]
Let's add a button at the top of /podcasts where the user can switch between "full view" and "compact view".  Currently, we have full view.

The new compact view should fit many episodes on one screen in a table format.  Episodes should look more like a row.  It should have a minimal height, small album art, photos from guest, title, Download (icon only, no text), play button, and "view more".  It should also have rate, and mark played.

In other words, remove the description and make everything compactly in one line.

If the title is too long for the screen, such as on mobile, then it should scroll like a marque.

Make sure that it is convenient for people to quickly rate episodes one by one.
